fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java

```diff
@@ -316,7 +326,7 @@ private static String stripUserAgentBaseURI(final String object) {
      *                    resource
      * @return a list of acl:Authorization objects
      */
-    private static List<WebACAuthorization> getAuthorizations(final FedoraResource aclResource,
+    private List<WebACAuthorization> getAuthorizations(final FedoraResource aclResource,
                                                               final boolean ancestorAcl) {
 
         final List<WebACAuthorization> authorizations = new ArrayList<>();
```

Comment:
Can you also remove the two lines *below* this one.
```java
//TODO figure out where the translator should be coming from
final IdentifierConverter<Resource, FedoraResource> translator = null;
```
I don't think they are needed anymore

Reply:
I removed it in the commits I have together for the `fcrepo-config.xml` PR. Is it okay to wait for that PR to remove?